config/samples/config_v1alpha1_verifier_dynamic.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: config.ratify.deislabs.io/v1alpha1
+kind: Verifier
+metadata:
+  name: verifier-dynamic
+spec:
+  name: dynamic
+  artifactTypes: application/vnd.ratify.spdx.v0
+  source:
+    artifact: index.docker.io/acanthamoeba/sample-plugin:v1
```

Comment:
This is used in tests, so it needs to point to a valid publicly accessible location. To start, this uses my personal Docker Hub account, but should point to an artifact alongside our other test images

@susanshi when we review this, can one of the maintainers push a plugin like we have done for images for the other e2e tests?

The steps would look something like:

```shell
CGO_ENABLED=0 go build -o=./sample ./plugins/verifier/sample
oras push wabbitnetworks.azurecr.io/test/sample-verifier-plugin:v1 ./sample
```